doc/pprof.md

```diff
@@ -235,10 +239,41 @@ them. This is useful to combine profiles from multiple processes of a
 distributed job. The profiles may be from different programs but must be
 compatible (for example, CPU profiles cannot be combined with heap profiles).
 
-pprof can subtract a profile from another in order to compare them. For that,
-use the **-base= _profile_** option, where *profile* is the filename or URL for the
-profile to be subtracted. This may result on some report entries having negative
-values.
+
+## Comparing profiles
+
+pprof can subtract a profile from another, provided the profiles are of 
+compatible types, in order to compare them. For that, use the 
+**-diff_base= _profile_** option, where *profile* is the filename or URL for the
+profile to be subtracted. This may result in some report entries having negative
+values.  Percentages in the report are relative to the total of 
+the diff base. 
+
+If the merged profile is output as a protocol buffer, all samples in
+the diff base profile will have a label with the key "pprof::base" and a value
+of "true". If pprof is then used to look at the merged profile, it will behave
+as if separate source and base profiles were passed in.
+
+The **-base** option can be used in the place of the **-diff_base** option, and
+may be preferrable when comparing two cumulative profiles, for example two
+contention profiles collected from the same program at different times. When 
+this flag is used and no negative values appear in the merged profile when
+aggregated at the address-level, percentages reported will be relative to the 
+the difference between the total for the source profile and the total for the
+base profile. In the general case, percentages will be relative to the total of
+the absolute value of all samples when aggregated at the address level.
+
+The **-normalize** option can be useful for determining the relative differences 
```

Comment:
Did we document -normalize elsewhere before? I'd make sure we don't double-document it now.

Reply:
We haven't documented normalize before.